### ISSUE TITLE
Install Pure Storage vSphere Client Plugin => Mismatch input variable in "Choose vCenter" task

### ISSUE THREAD

**Issue by m-garrido** [codyhosterman/vROInstallvSpherePlugin#1]
I try to use "Install Pure Storage vSphere Client Plugin" workflow from PS plugin 4.0.0 for vRO 8.2.
I think, the workflow don't set the proper input variable name for "Choose vCenter" task. This one pass the "vCenterSystem" empty VC:SdkConnexion variable but the task needs an array of VC:SdkConnexion. So, i presume we should pass "vCenterSystems" array variable instead. For now, at the begining of the task throw, an exception is raised because he can't take an element from empty VC:SdkConnexion variable.

**Comment by codyhosterman** [comment]
Yup this looks like a bug. Will fix it in 4.0.1. Thanks for reporting
